code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -183,21 +193,26 @@
 			new /obj/item/storage/belt/chameleon(src) // Unique but worth at least 2 tc
```

Comment:
The ninja stars are collectively worth whatever the throwing item box in the uplink is. 1tc iirc